src/EditorFeatures/VisualBasicTest/CodeActions/InlineTemporary/InlineTemporaryTests.vb

```diff
@@ -3831,5 +3831,53 @@ End Class
 
             Test(code, expected, compareTokens:=False)
         End Sub
+
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInlineTemporary)>
+        Public Sub TestSimpleConditionalAccess()
+            Dim code =
+<File>
+Class C
+    Sub M(args As String())
+        Dim [|x|] = args.Length.ToString()
+        Dim y = x?.ToString()
```

Comment:
Is it possible to have other forms of expressions like the following and does the fix work in those cases?
x?!dictionarykey
x?.<xmlelement>
x?...<xmldescendant> or 
x?.@xmlattribute